app/src/main/java/com/github/h3lp3rs/h3lp/SettingsActivity.kt

```diff
@@ -24,6 +30,25 @@ class SettingsActivity : AppCompatActivity() {
         loadSyncPref()
         findViewById<TextView>(R.id.unique_id_text).text = getUid()
         findViewById<TextView>(R.id.user_since_text).text = getCreationDate()
+
+        enforceSignInToCheck(R.id.medical_info_checkbox)
+        enforceSignInToCheck(R.id.user_cookie_checkbox)
+        enforceSignInToCheck(R.id.my_skills_checkbox)
```

Comment:
This is great! Why not do it in the personal profile and helper profile activities?

Reply:
Did not realise that the sign in was needed in those activities...

Reply:
Actually, isn't the filter for items applied locally? In that case it would work

Reply:
For the items, it makes sense as I didn't know the check was done locally. 
For the medical card, I mean it just doesn't make sense to me to have users create their profile if they're not signed in 🤷‍♀️ , the medical card is only used when users ask for help, and since guest users can't well it's just pointless imo, but it's not a big deal we can also let them create the profile and when they sign we push the data to the database as you said